types/promise-events/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for promise-events 0.1
+// Project: https://github.com/yanickrochon/promise-events#readme
+// Definitions by: Marcos Gutierrez <https://github.com/me>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export class EventEmitter {
```

Comment:
I think you want to extend EventEmitter from `events` as per https://github.com/yanickrochon/promise-events#api